src/background/services/tabEvents.ts

```diff
@@ -62,15 +63,16 @@ export class TabEvents {
     sender?: Runtime.MessageSender
   ) => {
     const { enabled } = await this.storage.get(['enabled'])
+    const t = tFactory(this.browser)
```

Comment:
Should we add this to the container directly? This way it will be available everywhere in the container if needed.

Reply:
`this.t()`?

Reply:
Yep, or `this.translate`.